modules/ob-installing-ko-strategy.adoc

```diff
@@ -0,0 +1,23 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ko-strategy_{context}"]
+= Installing ko strategy
+
+You can install the `ko` strategy at cluster level. By installing the `ko` strategy at cluster level, you can share it across different namespaces within your cluster.
```

Comment:
Same comment as above: "You can share the `ko` strategy across different namespaces within your cluster by installing it at cluster level."